resources/metadata/githubCreateIssue.yaml

```diff
@@ -33,6 +33,14 @@ spec:
         type: "[]string"
         default: []
         mandatory: false
+      - name: chunkSize
+        description: Defines size of the chunk. If content exceed chunk size it'll be sliced into chunks and stored in comments
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        type: int
+        default: 65500
```

Comment:
Should it be made clear what the maximum value is? Or what happens if you set `chunkSize` above it?